src/components/ButtonIcon/ButtonIcon.jsx

```diff
@@ -74,6 +90,7 @@ ButtonIcon.propTypes = {
   onClick: PropTypes.func,
   loading: PropTypes.bool,
   text: PropTypes.string,
+  tooltipText: PropTypes.string,
```

Comment:
I think tooltipPlacement should be added to the propTypes too, right?

Reply:
Sure!